pkg/metrics/api/api.go

```diff
@@ -0,0 +1,313 @@
+package api
+
+import (
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/openshift/installer/pkg/metrics/builder"
+	"github.com/openshift/installer/pkg/metrics/pushclient"
+)
+
+var (
```

Comment:
how many of these name const need to be public?

Reply:
the installer will use all of these const to create metrics.